tcl/org.testeditor.tcl.dsl/src/org/testeditor/tcl/dsl/validation/TclValidator.xtend

```diff
@@ -59,4 +74,62 @@ class TclValidator extends AbstractTclValidator {
 		}
 	}
 
-}
+	@Check
+	def checkVariableDerefUsage(TclModel tclModel) {
```

Comment:
Should have a return type (`void`) and be renamed to `checkVariableReferenceUsage`.